test/test_public_bindings.py

```diff
@@ -309,6 +309,7 @@ def test_modules_can_be_imported(self):
             "torch.onnx._internal.fx.op_validation",
             "torch.onnx._internal.fx.passes",
             "torch.onnx._internal.fx.passes._utils",
+            "torch.onnx._internal.fx.passes.assertion_removal",
```

Comment:
Adding new entries here is highly discouraged as it means that trying to "import" this module leads to a hard crash.
You should consider making the module importable from generic configs (in this case the based cpu image in CI).

Reply:
Thanks! I just found that the very same problem happened before: https://github.com/pytorch/pytorch/pull/117314, and what is happened is https://github.com/pytorch/pytorch/pull/117314#discussion_r1453867919. I wonder if there is a better way to do this. @xadupre @malfet